### PR TITLE
Add missing projects and update some existing ones

### PR DESCRIPTION
**Pull request by lukaszgryglicki**
cc @caniszczyk @castrojo

### CODE REVIEW

**Comment by lukaszgryglicki** [comment]
@castrojo build fails but I don't know why, I've fixed a few issues since 1st commit like extra `:` etc - but now it fails due to some strange errors that seem to be unrelated to my PR. Anyway, this needs a review, especially regarding categories/subcategories. Thanks!

**Comment by caniszczyk** [comment]
@AndreyKozlov1984 can you figure out what's broken here?

**Comment by castrojo** [comment]
I don't think the commented out logo snippets are going to work, the check is looking for actual logos afaict?

**Comment by AndreyKozlov1984** [comment]
I'll check how to fix this

**Comment by lukaszgryglicki** [comment]
I can't add real logos because there are no logos for those projects.

**Comment by caniszczyk** [comment]
@castrojo can you track down missing logos, they should be on github.com/cncf/artwork as part of project onboarding

**Comment by mrbobbytables** [comment]
Just an FYI - I added the missing projects here: https://github.com/cncf/landscape/pull/3373

**Comment by lukaszgryglicki** [comment]
Thanks, I saw & approved.

**Comment by lukaszgryglicki** [comment]
@caniszczyk @castrojo @AndreyKozlov1984 can you PTAL?
I've updated the PR - removed projects that were added in separate PRs since this PR was open - now it only contains changes needed on top of the current `master` branch.

Thanks!

**Comment by lukaszgryglicki** [comment]
Ping...

**Review by castrojo — APPROVED** [review]

**Comment by castrojo** [comment]
ok this looks fine to me but I'm still new so hoping for one more review!

File: landscape.yml [hunk 1]
```diff
@@ -3508,11 +3512,17 @@ landscape:
             crunchbase: https://www.crunchbase.com/organization/netflix
           - item:
             name: Xline
+            homepage_url: https://www.xline.cloud
```
Comment (castrojo) [inline landscape.yml:3515]:
```suggestion
```

**Review by castrojo — CHANGES_REQUESTED** [review]

File: landscape.yml (hunk 1 above)
Comment (lukaszgryglicki) [reply landscape.yml:3515]: Not quite sure what should I change?

Comment (lukaszgryglicki) [reply landscape.yml:3515]: Ohh sorry I see - this is a duplicated entry.

**Review by castrojo — APPROVED** [review]

**Pull request merged**